move/axelar_gateway/sources/versioned/gateway_v0.move

```diff
@@ -221,6 +229,18 @@ public(package) fun send_message(
     );
 }
 
+public(package) fun allow_function(self: &mut Gateway_v0, version: u64, function_name: String) {
```

Comment:
Add allow_function and disallow_function in version_control instead to avoid duplicating the logic